src/packagedcode/pypi.py

```diff
@@ -204,27 +223,51 @@ def parse_with_dparse(location):
         mode = 'r'
     with open(location, mode) as f:
         content = f.read()
-        df = dparse.parse(content, file_type=file_name)
-        df_dependencies = df.dependencies
-        if not df_dependencies:
-            return
-        package_dependencies = []
-        for df_dependency in df_dependencies:
-            specs = df_dependency.specs
-            requirement = None
-            if specs:
-                requirement = str(specs)
-            package_dependencies.append(
-                models.DependentPackage(
-                    purl=PackageURL(
-                        type='pypi', name=df_dependency.name).to_string(),
-                    scope='dependencies',
-                    is_runtime=True,
-                    is_optional=False,
-                    requirement=requirement,
-                )
+        
+    df = dparse.parse(content, file_type=dependency_type)
+    df_dependencies = df.dependencies
+
+    if not df_dependencies:
+        return
+
+    package_dependencies = []
+    for df_dependency in df_dependencies:
+        specs = list(df_dependency.specs._specs)
+        is_resolved = False
+        requirement = None
+        purl = PackageURL(
+            type='pypi',
+            name=df_dependency.name
+        ).to_string()
+        if specs:
+            requirement = specs[0]
+            if specs[0].operator == '==':
```

Comment:
what if there are more than one items in `specs`?

Reply:
It is fixed now